tests/acceptance/core/testcase/service.go

```diff
@@ -0,0 +1,85 @@
+package testcase
+
+import (
+	"github.com/k3s-io/k3s/tests/acceptance/core/service/assert"
+	"github.com/k3s-io/k3s/tests/acceptance/shared/util"
+	"github.com/onsi/ginkgo/v2"
+	"github.com/onsi/gomega"
+)
+
+func TestServiceClusterIp(g ginkgo.GinkgoTestingT, deployWorkload bool) {
+	if deployWorkload {
+		_, err := util.ManageWorkload("create", "clusterip.yaml", *util.Arch)
+		gomega.Expect(err).NotTo(gomega.HaveOccurred(),
+			"Cluster IP manifest not deployed")
+	}
+
+	err := assert.ValidateOnHost(util.GetClusterIp+util.KubeConfigFile, util.RunningAssert)
+	if err != nil {
+		ginkgo.GinkgoT().Logf("Error: %v", err)
+	}
```

Comment:
I mentioned this in another file, but it appears that every time we call `ValidateOnHost` we are only logging errors and don't actaully stop when they occur. Can you give me some context on why that is?